sphinx/ext/autosummary/generate.py

```diff
@@ -188,6 +189,29 @@ def get_members(obj, typ, include_public=[], imported=True):
                           if x in include_public or not x.startswith('_')]
                 return public, items
 
+            def get_package_members(out_dict, include_public=[], imported=True):
```

Comment:
Why did you not use return value? I feel `out_dict` is strange interface.

In addition, is `include_public` needed? It seems not given.

Reply:
Please add type annotations here.

Reply:
Type annotation added. `get_members` had an unused `include_public` so I added it to `get_package_members`  as well. If we remove it we should do it for both.

Reply:
No, `include_public` for `get_members() has been used. So we can't remove it.
https://github.com/sphinx-doc/sphinx/blob/4628264551a2c2ef16213abdd6965c7baf65c536/sphinx/ext/autosummary/generate.py#L205-L206